book/src/tutorial/preparation.md

```diff
@@ -0,0 +1,69 @@
+# Preparation
+In order to install gir and nicely structure the project, there are a few things to set up.
+
+## Set up the project folder
+In order to keep the project folder nicely organized, lets create a folder where we will work in and initialize the repo. We will create two library crates. pango will contain the safe wrapper crate and because it is a wrapper for the unsafe bindings, we create the pango-sys crate within the pango crate. If no Cargo.toml file is present in the sys create, a new one will be generated, so lets be safe and delete the automatically created file before we begin. The following commands will set up the project folder as described. 
```

Comment:
```suggestion
In order to keep the project folder nicely organized, let's create a folder where we will work in and initialize the repo. We will create two library crates. pango will contain the safe wrapper crate and because it is a wrapper for the unsafe bindings, we create the pango-sys crate within the pango crate. If no Cargo.toml file is present in the sys create, a new one will be generated, so lets be safe and delete the automatically created file before we begin. The following commands will set up the project folder as described. 
```